extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/hll/sql/HllSketchSqlAggregatorTest.java

```diff
@@ -300,6 +307,15 @@ public SpecificSegmentsQuerySegmentWalker createQuerySegmentWalker(
                      .size(0)
                      .build(),
           index
+      ).add(
+          DataSegment.builder()
+                     .dataSource(CalciteTests.WIKIPEDIA_FIRST_LAST)
+                     .interval(Intervals.of("2015-09-12/2015-09-13"))
+                     .version("1")
+                     .shardSpec(new NumberedShardSpec(0, 0))
+                     .size(0)
+                     .build(),
+          TestDataBuilder.makeWikipediaIndexWithAggregation(tempDirProducer.newTempFolder())
```

Comment:
Instead of this, we can use the parent method to register the datasource:

```java

    public SpecificSegmentsQuerySegmentWalker createQuerySegmentWalker(
        final QueryRunnerFactoryConglomerate conglomerate,
        final JoinableFactoryWrapper joinableFactory,
        final Injector injector
    )
    {
        SpecificSegmentsQuerySegmentWalker walker = super.createQuerySegmentWalker(conglomerate...);
        // Do the remaining processing that's done earlier here
     }

Reply:
In the `remaining processing that's done earlier here` part, we're using `CalciteTests.DATASOURCE1` as the datasource name. Calling super() ends up conflicting with that, and fails all the tests. So we'd have to use a different name for the datasource throughout this file, which would be a good bit of unrelated changes to this PR. Hence didn't go that route. Hope that works!